### PR TITLE
Doc update ypise

### PR DESCRIPTION
**Pull request by ypise**
### Changes
 - [x] Why is this change required? What problem does it solve?
         Updates the infrastructure requirements in deployment document  
 - [ ] If proposing a new change then please raise an issue first

### How Has This Been Tested? (Optional)  
 - [ ] Please describe in detail how you tested your changes.  
        These have been tested on a VM environment  
 - [ ] Include details of your testing environment, and the tests you ran to  
        The tests were performed on Lab VMs  
 - [ ] How your change affects other areas of the code, etc.  
        No impacts. The change would improvise the performance and stability of deployments.  

### Checklist
 - [x] tested locally
 - [ ] added new dependencies
 - [x] updated the docs
 - [ ] added a test

### CODE REVIEW

**Review by mukul-seagate11 — APPROVED** [review]
looks good

File: doc/ProvisionReleaseBuild.md [hunk 1]
```diff
@@ -6,34 +6,35 @@ You will need to complete this [guide](https://github.com/Seagate/cortx/blob/mai
...
- -  [x]  Please create VM(s) with at least 1 CPUs and 4GB of RAM.  
- -  [x]  For single-node VM deployment, ensure the VM is created with 2+ attached disks.
- -  [x]  Do you see the devices on execution of this command: lsblk ?  
- -  [x]  Do the systems on your setup have valid hostnames, are the hostnames accessible: ping <hostname>?  
- -  [x]  Do you have IPs' assigned to all NICs ensxx, ensxx and ensxx?  
- -  [x]  Identify primary node and run below commands on primary node  
-    
-**NOTE**: For single-node VM, the VM node itself is treated as primary node.
+ -  [ ]  Please create VM(s) with at least 4 CPUs and 4GB of RAM.  
+ -  [ ]  For single-node VM deployment, ensure the VM is created with 8+ attached disks in addition to at least 2 ide storage for OS.
```
Comment (johnbent) [inline doc/ProvisionReleaseBuild.md:10]: I am not comfortable with 8+ drives.  This isn't necessary for community where @hessio has successfully demonstrated that it works with 1+0.  @ypise , is it possible to leave this as 2+ as it was before?

**Review by johnbent — CHANGES_REQUESTED** [review]

File: doc/ProvisionReleaseBuild.md (hunk 1 above)
Comment (ypise) [reply doc/ProvisionReleaseBuild.md:10]: @johnbent,
I increased it to 8+ as it's necessary for 4+2+2 config in Motr. In one of arch discussions, it was mentioned that the erasure coding was core feature of our offering and community should be able to test it.

I'll revert it back to 2+.

Comment (johnbent) [reply doc/ProvisionReleaseBuild.md:10]: Hey Yash.  Erasure feature is indeed an awesome feature and we should make it available to the community.  However, it shouldn't be required I don't think.  Would it be possible to have the instructions say:

*   If you only have 1 disk, do ....
*   If you have 8 disks, you can try erasure.  Do .....

That would be the very best.

Comment (ypise) [reply doc/ProvisionReleaseBuild.md]: Sure John, I've incorporated this suggestion in the following commit:
https://github.com/Seagate/cortx/pull/1081/commits/e9e2664d79ec7f8ac2eae36bc3cfb0b2031c55a8

**Comment by ypise** [comment]
> This is awesome!  Thanks @ypise !

You are welcome! :)

**Comment by mukul-seagate11** [comment]
@ypise, can you update your branch so that the PR can be merged

File: doc/ProvisionReleaseBuild.md [hunk 2]
```diff
@@ -6,34 +6,39 @@ You will need to complete this [guide](https://github.com/Seagate/cortx/blob/mai
...
 
- -  [x]  Please create VM(s) with at least 1 CPUs and 4GB of RAM.  
- -  [x]  For single-node VM deployment, ensure the VM is created with 2+ attached disks.
- -  [x]  Do you see the devices on execution of this command: lsblk ?  
- -  [x]  Do the systems on your setup have valid hostnames, are the hostnames accessible: ping <hostname>?  
- -  [x]  Do you have IPs' assigned to all NICs ensxx, ensxx and ensxx?  
- -  [x]  Identify primary node and run below commands on primary node  
-    
-**NOTE**: For single-node VM, the VM node itself is treated as primary node.
+ -  [ ]  Please create VM(s) with at least 4 CPUs and 4 GB of RAM
```
Comment (mukul-seagate11) [inline doc/ProvisionReleaseBuild.md:9]: also, we can update to 2 vCPUs which is sufficient

**Review by johnbent — APPROVED** [review]

**Pull request merged**